core-bundle/contao/themes/flexible/styles/basic.css

```diff
@@ -927,28 +946,69 @@ a.tl_submit {
 	font-weight: 600;
 }
 
-@media (-webkit-min-device-pixel-ratio: 2), (min-resolution: 192dpi) {
-	.badge-title {
-		font-weight: 500;
+/* Tablets */
+@media (min-width: 768px) {
+	.widget-group {
+		--widget-gap-cols: 30px;
+	}
+
+	.w16, .w25, .w33, .w50, .w75 {
+		--widget-cols: 6;
+	}
+
+	.m12 {
+		padding: 18px 0 16px;
+	}
```

Comment:
Could we omit this entirely if we made `align-items: center;` the default behavior?

Reply:
> Could we omit this entirely if we made `align-items: center;` the default behavior?

We can't, because:
- it would have that annoying 1px offset, hence needing to remove 1px padding from `.tl_box h3, .tl_tbox h3, .tl_xpl h3`
   ![image](https://github.com/contao/contao/assets/55794780/fcb04d66-90ef-47ba-ae2b-f61fda578920)

- if you don't have a label, it would never be centered:
   ![image](https://github.com/contao/contao/assets/55794780/84e6bee4-8843-499d-8f20-2e0c8785f527)

Reply:
> if you don't have a label, it would never be centered:

imho we can ignore that

Reply:
So we reduce 1px on the tlbox label, then that should be fine as well with the offsez

Reply:
Or we add a 1px top margin to the checkbox containers?

Reply:
With https://github.com/zoglo/contao/pull/2 it works without the `.m12` class for me.

Reply:
When `.widget-group` isn't present, it would still needs a fallback:
See: 0e6e8ff1974f966430a7577ae565e770c9f86dff